### ISSUE TITLE
2.8.5 Patch Release Plan (security release)

### ISSUE THREAD

**Issue by joeparsons** [az-digital/az_quickstart#3159]
- #3155

**Comment by joeparsons** [comment]
## Pick list
```
git cherry-pick 061e2d8118540dc02f9bd276a6af8187e8de193b
```

## Release notes snippet
`git log 2.8.4..HEAD --pretty='format:- %s'`
```
_Note: This release contains a security update for the Migrate Tools module addressing a moderately critical security vulnerability ([SA-CONTRIB-2024-008](https://www.drupal.org/SA-CONTRIB-2024-008))._

- Closes #3158: Update drupal/migrate_tools requirement from 6.0.2 to 6.0.3 for SA-CONTRIB-2024-008 (#3155)
- Back to dev.
```

**Comment by joeparsons** [comment]
https://github.com/az-digital/az_quickstart/releases/tag/2.8.5
